packages/angular_devkit/build_angular/package.json

```diff
@@ -16,9 +16,9 @@
     "@ngtools/webpack": "0.0.0",
     "ajv": "6.10.2",
     "autoprefixer": "9.6.1",
-    "browserslist": "4.8.3",
+    "browserslist": "4.8.6",
```

Comment:
Nit: probably we should unpin these as discussed here https://github.com/angular/angular-cli/issues/16790

Reply:
I think we should probably only do that on master to prevent any accidental breakage.
Also, `caniuse-lite@1.0.30001025` appears to cause the same failure so it needs to be pinned until the issue is resolved upstream.

Reply:
Good point